bindata/network/ovn-kubernetes/monitor.yaml

```diff
@@ -76,9 +76,9 @@ spec:
   publishNotReadyAddresses: true
   ports:
   - name: metrics
-    port: 9101
+    port: 9103
```

Comment:
(this could be done as a separate PR...)

Reply:
Will do.